src/Lib/Server/Auth.hs

```diff
@@ -36,24 +36,26 @@ instance FromJSON LoginRequest
 instance ToJSON LoginRequest
 
 newtype LoginResponse = LoginResponse
-  { loginResponseToken :: Text
+  { loginResponseToken :: JwtToken
   } deriving (Generic, Show, Eq)
 
 instance ElmType LoginResponse
 instance FromJSON LoginResponse
 instance ToJSON LoginResponse
 
+instance ElmType JwtToken
```

Comment:
It's better to move this instance to `Lib.Core.Jwt` module to avoid orphan instances warning.